CONTRIBUTING.md

```diff
@@ -67,6 +67,23 @@ Most components should use the compose framework as it offers the comprehensive
    1. `<new-component>.<types | settings | platform | blah>.tsx` (Optional)
       - Optional extra files to subdivide your code however you see fit. You can also add platform specific files as you see fit.
 
+### Packaging your component (or make it into an importable module)
+
+1. Delete the cache and run `yarn` to get the lib and node_module folders to auto install
+   1. From root folder: `watchman watch-del-all`
+   1. `rm -rf node_modules`
+   1. `yarn`
+   1. `yarn build`
+   1. `cd apps/ios/src`
+   1. `pod install` since there are Apple components that need to be reconnected
+   1. Restart VSCode/Editor
+   1. Close packager
+   1. Close simulator
+   1. `cd apps/ios`
+   1. `yarn ios`
+1. Add your new module as a dependency in `apps/fluent-tester/package.json`
+1. Note: Make sure that all naming is consistent: the component you are exporting, the file name you are exporting, and what you are importing
```

Comment:
I'm not sure we actually need _all_ of these steps. I think this can be condensed to something like: "Add your new package to `apps/fluent-tester/package.json` so that we can import and test it, and run `yarn && yarn build`. If you your component has native apple code, remember to run `pod install`."

Reply:
I just added all of the steps that I did to get it working, but it looks like deleting and restarting everything isn't needed. Is there somewhere else where the "deleting cache" steps could go? Sometimes bugs go away once things are reset.

Reply:
I've seen the metro bundler spit out these steps whenever I run into an error running it, so I think that's something that is easiest enough to find yourself.

You could put it in a new troubleshooting section too if you like.